src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -58,7 +58,7 @@ class RecallControllerTest {
   private val prisonValidationService = mockk<PrisonValidationService>()
   private val courtValidationService = mockk<CourtValidationService>()
   private val tokenExtractor = mockk<TokenExtractor>()
-  private val now = OffsetDateTime.now()
+  private val now = OffsetDateTime.now(ZoneId.of("UTC"))
```

Comment:
Must have accidentally reversed this (fix for BST within IntelliJ)

Reply:
IntelliJ -> your laptop -> your settings?  IMO it should be fixable by making sure our environments mimic CI and deployed environments for everything locale related?